hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -240,12 +240,6 @@ public class HoodieWriteConfig extends HoodieConfig {
       .withDocumentation("Schema string representing the latest schema of the table. Hudi passes this to "
           + "implementations of evolution of schema");
 
-  public static final ConfigProperty<Boolean> ENABLE_INTERNAL_SCHEMA_CACHE = ConfigProperty
-      .key("hoodie.schema.cache.enable")
-      .defaultValue(false)
-      .markAdvanced()
-      .withDocumentation("cache query internalSchemas in driver/executor side");
```

Comment:
Thanks for the contribution, do you check the eviction strategy already? Should we have a tweak for it after the removing of the option?

Reply:
Yes, i've checked the eviction strategy and I think we don't need any tweaks. The cache is instantiated with maximumSize(1000) parameter, this means (from Caffein docs):  
`Specifies the maximum number of entries the cache may contain. Note that the cache may evict
   * an entry before this limit is exceeded or temporarily exceed the threshold while evicting.
   * As the cache size grows close to the maximum, the cache evicts entries that are less likely to
   * be used again. For example, the cache may evict an entry because it hasn't been used recently
   * or very often.`

Reply:
Yes, i've checked the eviction strategy and I think we don't need any tweaks. The cache is instantiated with maximumSize(1000) parameter, this means (from Caffein docs):  
`Specifies the maximum number of entries the cache may contain. Note that the cache may evict
   * an entry before this limit is exceeded or temporarily exceed the threshold while evicting.
   * As the cache size grows close to the maximum, the cache evicts entries that are less likely to
   * be used again. For example, the cache may evict an entry because it hasn't been used recently
   * or very often.`

Reply:
Okay, let's resolve the test failures.